### PR TITLE
React Calculator: First milestone

### PR DESCRIPTION
**Pull request by SmithVyne**
The purpose of this first milestone is to get the initial setup for the project ready.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (christianotieno) [inline .eslintrc.json:1]: ### FIX linter errors.
- [ ] Fix the informing linter errors currently present in the PR of this branch before we proceed.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "arrow-parens": ["error", "as-needed"]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
```
Comment (christianotieno) [inline .eslintrc.json:27]: ### Remove unnecessary files
- [ ] While at it, please make sure unnecessary files or assets, such as the React SVG Logo, get deleted before we proceed as well. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/)).

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
```
Comment (christianotieno) [inline .eslintrc.json:28]: ### ADD relevant info about the project in the README
- [ ] Make sure the project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

- [ ] Make sure the app is deployed on `Heroku` and the live link is included in the README file.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/app-deployed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

**Review by christianotieno — CHANGES_REQUESTED** [review]
### Changes Required ♻️ 
Hello @SmithVyne 👋 
Congratulations with the milestone so far. Everything is coming together neatly.

Thank you for initialising the app with  `create-react-app` as instructed, and adding all the required dependencies in their right environment as well. Keep it up 👏 

Before we proceed, we need to take care of the following instances to make this milestone more solid and get you ready for what's ahead `(Goal is to make the project easier to build)` 👇 

### Happy coding 💻 

#### Christian Otieno (TSE)

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @SmithVyne  👋

**Good job so far here. 💪**


- [ ] However, as stated in the previous review kindly add a more descriptive readme.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix this issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**